services/tenant-ui/frontend/src/components/profile/issuance/Endorser.vue

```diff
@@ -1,12 +1,91 @@
-      :disabled="hasEndorserConn"
-      @change="connectToEndorser"
-    />
-
+  <div v-if="canBecomeIssuer" class="my-1">
+    <DataTable
+      v-model:filters="filter"
+      :loading="loading"
+      :value="formattedLedgers"
+      :paginator="false"
+      :rows="TABLE_OPT.ROWS_DEFAULT"
+      :rows-per-page-options="TABLE_OPT.ROWS_OPTIONS"
+      selection-mode="single"
+      data-key="ledger_id"
+      sort-field="ledger_id"
+      :sort-order="-1"
+      filter-display="menu"
+    >
+      <template #header>
+        <div class="flex justify-content-between">
+          <div class="flex justify-content-start"></div>
+          <div class="flex justify-content-end">
+            <span class="p-input-icon-left">
+              <i class="pi pi-search ml-0" />
+              <InputText
+                v-model="filter.global.value"
+                placeholder="Search Endorsers"
+              />
+            </span>
+          </div>
+        </div>
+      </template>
+      <template #empty>{{ $t('common.noRecordsFound') }}</template>
+      <template #loading>{{ $t('common.loading') }}</template>
+      <Column :sortable="false" header="Actions">
+        <template #body="{ data }">
+          <span v-if="isLedgerSet && data.ledger_id === currWriteLedger">
+            <i class="pi pi-check-circle p-tag-success"></i>
+          </span>
+          <span
+            v-if="
+              !isLedgerSet ||
+              (isLedgerSet &&
+                enableLedgerSwitch &&
+                data.ledger_id !== currWriteLedger)
+            "
+          >
+            <Button
+              :label="$t('profile.connectToEndorserAndRegisterDID')"
+              icon="pi pi-check-square"
+              class="p-button-rounded p-button-icon-only p-button-text"
+              @click="connecttoLedger(data.ledger_id)"
+            />
+          </span>
+        </template>
+      </Column>
+      <Column
+        :sortable="true"
+        field="ledger_id"
+        header="Ledger Identifier"
+        filter-field="ledger_id"
+        :show-filter-match-modes="false"
+      >
+        <template #filter="{ filterModel, filterCallback }">
+          <InputText
+            v-model="filterModel.value"
+            type="text"
+            class="p-column-filter"
+            placeholder="Search By Ledger"
+            @input="filterCallback()"
+          />
+        </template>
+      </Column>
+      <Column
+        :sortable="true"
+        field="endorser_alias"
+        header="Endorser Alias"
+        filter-field="endorser_alias"
+        :show-filter-match-modes="false"
+      >
+        <template #filter="{ filterModel, filterCallback }">
+          <InputText
+            v-model="filterModel.value"
+            type="text"
+            class="p-column-filter"
+            placeholder="Search By Endorser"
+            @input="filterCallback()"
+          />
+        </template>
+      </Column>
+    </DataTable>
```

Comment:
In the `div` surrounding the accordion below this (line 97) let's add `class="mt-3"`